app/templates/macros/forms.html

```diff
@@ -0,0 +1,79 @@
+{% macro question_input(name, label, value='', hint='', errors=None) -%}
```

Comment:
Are these macros the kind of thing that should live in the Frontend Toolkit? This pull-request from @quis has at least some of these elements in it: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/70